.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         ruby_version: ['2.7', '3.0', '3.1']
-        rails_gemfile: ['6.0', '6.1']
+        rails_gemfile: ['6.0', '6.1', '7.0']
```

Comment:
6.0 is reaching its end of life soon but again if it's not annoying anyone we should keep it. https://endoflife.date/rails